apps/jest-expo-mock-generator/App.js

```diff
@@ -1,10 +1,17 @@
 import mux from '@expo/mux';
+import { setStringAsync } from 'expo-clipboard';
 import Constants from 'expo-constants';
 import getInstallationIdAsync from 'expo/build/environment/getInstallationIdAsync';
 import React from 'react';
-import { NativeModules, StyleSheet, Text, View } from 'react-native';
+import { Button, NativeModules, StyleSheet, Text, View } from 'react-native';
 import { v4 as uuidV4 } from 'uuid';
 
+// A workaround for `TypeError: Cannot read property 'now' of undefined` error.
+
+global.performance = {
```

Comment:
```suggestion
// A workaround for `TypeError: Cannot read property 'now' of undefined` error.
global.performance = {
```
Also for more context it might be worth adding what throws that error 🤔